### DISCUSSION TITLE
"Types" for CodeProject.ai

### DISCUSSION THREAD

**Discussion by mkammes** [djdd87/SynoAI#174] [General]
Hello!

There are a finite number of "Types" to define when using Deepstack. What Types are available when using CodeProject.ai-server?  I'm unable to find a list.  Thanks!

**Comment by mkammes** [comment]
Cancel.  Found this:

The object detection module uses YOLO (You Only Look Once) to locate and classify the objects the models have been trained on. At this point there are 80 different types of objects that can be detected:

- person
- bicycle, car, motorbike, aeroplane, bus, train, truck, boat
- traffic light, fire hydrant, stop sign, parking meter, bench
- cat, dog, horse, sheep, cow, elephant, bear, zebra, giraffe
- backpack, umbrella, handbag, tie, suitcase, frisbee, skis, snowboard, sports ball, kite, baseball bat, baseball glove, skateboard, surfboard, tennis racket
- bottle, wine glass, cup, fork, knife, spoon, bowl
- banana, apple, sandwich, orange, broccoli, carrot, hot dog, pizza, donut, cake
- chair, sofa, pottedplant, bed, diningtable, toilet, tvmonitor, laptop, mouse, remote, keyboard, cell phone, microwave, oven, toaster, sink, refrigerator, book, clock, vase, scissors, teddy bear, hair drier, toothbrush

**Reply by djdd87** [reply]
Thanks, I've logged an issue to add this to the wiki.
